build/ci/cloudbuild.push_image.yaml

```diff
@@ -20,8 +20,17 @@ steps:
       - -c
       - |
         set -e
-        DOCKER_BUILDKIT=1 docker build -f build/web_server/Dockerfile --tag gcr.io/$_PROJECT_ID/datacommons-website:${_TAG} .
+        DOCKER_BUILDKIT=1
+
+        # Website container
+        docker build -f build/web_server/Dockerfile --target server --tag gcr.io/$_PROJECT_ID/datacommons-website:${_TAG} .
         docker push gcr.io/$_PROJECT_ID/datacommons-website:${_TAG}
+
+        # Bundle container with website, esp and mixer.
+        docker build -f build/web_server/Dockerfile  --target website-compose --tag gcr.io/$_PROJECT_ID/datacommons-website-compose:${_TAG} .
+        docker push gcr.io/$_PROJECT_ID/datacommons-website-compose:${_TAG}
+        docker build -f build/web_server/Dockerfile  --target website-compose --tag gcr.io/$_PROJECT_ID/datacommons-website-compose:latest .
+        docker push gcr.io/$_PROJECT_ID/datacommons-website-compose:latest
   - id: push-nl-server
```

Comment:
nit: how much time does this add to autopush? :)

Reply:
Actually not much from https://pantheon.corp.google.com/cloud-build/builds;region=global/344448d5-2ed0-4a37-ba33-03c2b0c99b96;step=0?mods=-monitoring_api_staging&project=datcom-ci. Since most stages are cached.